source/javascripts/config.js

```diff
@@ -0,0 +1,29 @@
+(function () {
+  "use strict";
+
+  var CONFIG = {
+    "_default": {
+      apiBaseURL: "https://api.remotelock.com",
+      oAuthBaseURL: "https://connect.remotelock.com",
+      helpLinkURL: "https://support.remotelock.com",
+      zendeskWidgetKey: ""
+    },
+    "remotelock.com": {
+      apiBaseURL: "https://api.remotelock.com",
+      oAuthBaseURL: "https://connect.remotelock.com",
+      helpLinkURL: "https://support.remotelock.com",
+      zendeskWidgetKey: ""
+    }
```

Comment:
I like the idea of default. But since we want to use the default RL values unless overridden, I'd suggest this:

```
    "_default": {
      apiBaseURL: "https://api.remotelock.com",
      oAuthBaseURL: "https://connect.remotelock.com",
      helpLinkURL: "https://support.remotelock.com",
      zendeskWidgetKey: "6beb78d2-be02-4baf-81ec-779cba7f7533"
    },
    "remotelock.jp": {
      apiBaseURL: "https://api.remotelock.jp",
      oAuthBaseURL: "https://connect.remotelock.jp",
      helpLinkURL: "https://remotelocksupport.kke.co.jp",
      zendeskWidgetKey: "2951fe28-8c55-453d-ab6a-bc22b135c626"
    }
```

c/c @scnfilho

Reply:
Ok, I'll go with @scnfilho suggestion to remove fetch domain logic and just return the porperties, eg: 

`(function () {
  "use strict";

  window.config = {
    apiBaseURL: "https://api.remotelock.com",
    oAuthBaseURL: "https://connect.remotelock.com",
    helpLinkURL: "https://support.remotelock.com",
    zendeskWidgetKey: "6beb78d2-be02-4baf-81ec-779cba7f7533"
  };
})();`

**But I want to confirm if we should commit this key.**

Reply:
I was writing this and didn't saw the other response!